Rytis_George_Baltakys/Javascript/isItMyBirthday.js

```diff
@@ -0,0 +1,24 @@
+// Birthday Countdown
+
+function isItMyBirthdayYet(day) {
+
```

Comment:
This function would be a great opportunity to use a 'switch' statement.  Check this out...
http://www.w3schools.com/js/js_switch.asp